### PR TITLE
Create Notice Widget

### PR DESCRIPTION
**Pull request by vwbaker**
In a future PR, I will apply this to the notice boxes that we have for
inspections and stackunwinding.

Bug: http://b/240112046

### CODE REVIEW

File: src/UtilWidgets/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
```
Comment (beckerhe) [inline src/UtilWidgets/CMakeLists.txt:1]: Nit: 2022

File: src/UtilWidgets/include/UtilWidgets/NoticeWidget.h [hunk 2]
```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef UTIL_WIDGETS_NOTICE_WIDGET_H_
+#define UTIL_WIDGETS_NOTICE_WIDGET_H_
+
+#include <QWidget>
```
Comment (beckerhe) [inline src/UtilWidgets/include/UtilWidgets/NoticeWidget.h:8]: Can you also include <string>, <memory> and <QPaintEvent> to make this header self contained?

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/UtilWidgets/NoticeWidget.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```
Comment (antonrohr) [inline src/UtilWidgets/NoticeWidget.cpp]: 2022 here and the other files as well

**Review by antonrohr — APPROVED** [review]
LGTM!

File: src/UtilWidgets/NoticeWidget.cpp (hunk 3 above)
Comment (antonrohr) [reply src/UtilWidgets/NoticeWidget.cpp]: too late

File: src/UtilWidgets/include/UtilWidgets/NoticeWidget.h (hunk 2 above)
Comment (vwbaker) [reply src/UtilWidgets/include/UtilWidgets/NoticeWidget.h:9]: Done.

File: src/UtilWidgets/CMakeLists.txt (hunk 1 above)
Comment (vwbaker) [reply src/UtilWidgets/CMakeLists.txt]: Done.

**Pull request merged**